### PR TITLE
feat/8/right-user-reviews (#14)

### PR DESCRIPTION
**Pull request by prabhanukash**
- introduced reviewsList component
- used shadcn components( Card, Spinner, Pagination, Avatar, Box, Button, Flex)
  - included pagination for list of reviews
  - added loading during navigation of pages & and initial loading
  - added avatar for the stores for which reviews have given
  - added profile with stats component
  
  
  design for the right panel user reviews - https://excalidraw.com/#room=4d1cbd422407c74b9045,MZ6rec2n6nzmW1tMjNezlA

### CODE REVIEW

File: tailwind.config.ts
Comment (Talhaaa99) [inline tailwind.config.ts:1]: this file might need to be removed, we've got 2 tailwind config files, one is js and one in ts

Comment (prabhanukash) [reply tailwind.config.ts:1]: removed

**Comment by verschaevesiebe** [comment]
Resolve the merge conflicts please

**Comment by verschaevesiebe** [comment]
@prabhanukash  can you resolve eslint errors ? i also see that the right column pops an 500 error.

image

Map also doesn't load.

**Comment by prabhanukash** [comment]
image

**Review by Talhaaa99 — APPROVED** [review]
looks good

**Pull request merged**